tarantool/connection.py

```diff
@@ -623,7 +624,7 @@ def __init__(self, host, port,
             Unix sockets.
         :type host: :obj:`str` or :obj:`None`
 
-        :param port: Server port or Unix socket path.
+        :param port: Server port, socket fd or Unix socket path.
```

Comment:
Maybe it would be clearer to make a separate parameter?

Reply:
Well, we had decided to add a separate parameter